pucauto.py

```diff
@@ -317,6 +317,61 @@ def find_highest_value_bundle(trades):
         return None
 
 
+def optimal_bundle(highest_value_bundle):
+    """The best bundle if you have more tradable cards than the other guy has
+    points is the group of cards that comes closest to reaching their 
+    available points.
+
+    This does the old sort from complete_trades prior to this commit if there
+    is less value than the reciever has available points.
+
+    Otherwise, it finds the biggest bundle under their available poits, sorts 
+    that by card value, then sorts the leftovers by card value just in case 
+    one of the things you're trying to trade out gets jumped on by a third 
+    party."""
+
+
+
+    cards = highest_value_bundle[1]["cards"]
+    sorted_cards = []
+
+
+    
+    if highest_value_bundle[1]["value"] <= highest_value_bundle[1]["points"]:
+        sorted_cards = sorted(cards, key=lambda k: k["value"], reverse=True)
+    else:
+        largest = 0
+        sorted_cards = []
+        for i in range (1, len(cards)):
+            can_change = False
+            for j in itertools.combinations(cards, i):
+                bundle_value = 0
+                for k in j:
+                    bundle_value += k["value"]
+                if bundle_value <= highest_value_bundle[1]["points"]:
```

Comment:
Should this be just `<` instead of `<=`. If a combination (aka bundle_value) is equal to the number of points the user has, does the loop need to iterate again?